src/iterative_ensemble_smoother/experimental.py

```diff
@@ -177,6 +139,47 @@ def _corr_to_cov_inplace(
         corr_XY *= stds_Y[np.newaxis, :]
         return corr_XY
 
+    @classmethod
```

Comment:
you could use @staticmethod instead of @classmethod since you do not use cls for anything

Reply:
Good point, thanks. I've pushed a fix.